### DISCUSSION TITLE
pickui savefig

### DISCUSSION THREAD

**Discussion by SatisfactionLow** [xumi1993/seispy#72] [General]
it seems that if I didn't preview the figure, saving the image would result error "Nonetype has no attribute savefig.

![image](https://github.com/xumi1993/seispy/assets/129023056/a06b2689-47dc-4320-b043-9304b9052694)

changing line 113 in pickui.py to take out "not" can generate preview as I save, but im not sure if this would cause any other problem.

![image](https://github.com/xumi1993/seispy/assets/129023056/e9629b12-c00f-43e7-9559-aa54972f4b56)

**Comment by xumi1993** [comment]
Hi, I'm sorry about that and thank you for debugging. May I know the version you used, because in my test cases, it works fine. Or you can send you results (RFs) to me. I will recurrent this issue on my side.

**Reply by SatisfactionLow** [reply]
i'm using the latest 1.3.4 version, but because i was changing stuff also. So if you cannot recurrent then probrably it's me accidently causing this issue.

**Reply by xumi1993** [reply]
Hi, I've made some tests on my side, but everything is okay. Could you reset your version to 1.3.4 and test it again? Please let me know if this issue is still raised.
